webapp/webapp.spec.yaml

```diff
@@ -741,6 +741,23 @@ components:
         modified_since:
           type: string
           example: '2018-04-05T01:23:45+02:00'
+        type:
+          oneOf:
+            - type: string
+              example: 'security'
+            - type: array
+              items:
+                type: string
+                example: security
+        severity:
+          oneOf:
+            - type: string
+              example: 'Critical'
```

Comment:
shouldn't this be nullable as well?

Reply:
hmm, I fixed it
but I have tested it before and it was working fine 🤷‍♂

Reply:
are you testing it with upgraded schema (null severity_id columns in errata table)? and current webapp dump (generated by reposcan)?